operations/serviceorders.md

```diff
@@ -0,0 +1,75 @@
+# Service orders
+
+## Get all service order notes
+
+Returns all notes associated with the service order. This operation uses [Pagination](../guidelines/pagination.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/serviceOrderNotes/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ServiceOrderIds": [
+        "1dc0c6bf-2ce6-4a9f-af97-af5c01676720"
+    ],
+    "ServiceOrderNoteIds": [
+        "a06a225b-00f7-48c8-a463-af5c016768e9"
+    ],
+    "Limitation": { "Count": 1 }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceOrderIds` | array of string | required, max 1000 items | Unique identifiers of [Service order](#service-order). |
+| `ServiceOrderNoteIds` | array of string | optional, max 1000 items | Unique identifiers of [Service order notes](#service-order-note). Use this property if you want to fetch specific service order note. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of Service order notes returned. |
+
+### Response
+
+```javascript
+{
+    "ServiceOrderNotes": [
+        {
+            "Id": "a06a225b-00f7-48c8-a463-af5c016768e9",
+            "ServiceOrderId": "1dc0c6bf-2ce6-4a9f-af97-af5c01676720",
+            "Text": "This should be an order note, right?",
```

Comment:
```suggestion
            "Text": "Shaken, not stirred.",
```
Kinda tricky what to put here as an example... what about this?

Reply:
I just put in a text from my response within my localhost lmao 😆